tests/adapter/conftest.py

```diff
@@ -65,17 +65,3 @@ def parsed_gene():
         "build": "37",
     }
     return gene_info
-
-
-@pytest.fixture
```

Comment:
There is already another fixture named `test_omim_term` that can be used in all tests. Removing this one